### ISSUE TITLE
When connecting using AZ CLI authentication, the interactive property is overridden + cache not being generated

### ISSUE THREAD

**Issue by FaresKi** [mattwar/Kusto.Toolkit#11]
Hi!
I got a code similar to the following:
```csp
KustoConnectionStringBuilder connection = new KustoConnectionStringBuilder(cluster, "DB").WithAadAzCliAuthentication(false);
var loader = new CachedSymbolLoader(connection.ToString(), $"{Directory.GetCurrentDirectory()}/cache");
var databaseSchema = await loader.LoadDatabaseAsync("DB", null, true);
```
Despite disabling the `interactive` property in `WithAadAzCliAuthentication`, I still got my browser opened and requesting confirmation. 
Also, my cache isn't being generated in my folder as expected.
btw, I'm on MacOS, if that helps.

TIA!

**Comment by mattwar** [comment]
Does your cluster name or database name have characters that would be invalid as part of file/path name on mac os?

**Comment by mattwar** [comment]
To help with the authentication problem, can you try creating the ServerSymbolLoader with the KustoConnectionStringBuilder as a constructor argument, and then created the CachedSymbolLoader with that ServerSymbolLoader as an argument?  When you call ToString on it the conneciton string builder, it strips out any credential information it has before returning the string.  

This may be the problem.

**Comment by mattwar** [comment]
I'm adding an overload that takes the connection builder to help avoid this problem.

**Comment by mattwar** [comment]
I've published a new version 1.3.2 to nuget with these changes.

**Comment by mattwar** [comment]
We can reopen this issue if the changes did not solve the problem.
